### PR TITLE
Feature/fix static files with auth

### PR DESCRIPTION
**Pull request by franmoore05**
Updating the static files with auth stack so that it is possible to publish a collection and use the static files functionality.

To test, please clone this branch and cd in to the /v2/static-files-with-auth directory. Follow the instructions in the readme to set cognito variables in the local.env file and run the command 

```go
make up
``` 

All containers should run.

### CODE REVIEW

File: v2/manifests/core-ons/zebedee.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ services:
       OTEL_JAVAAGENT_ENABLED:         ${OTEL_JAVAAGENT_ENABLED:-"false"}
       ELASTIC_SEARCH_CLUSTER:         "cluster"
       ELASTIC_SEARCH_SERVER:          "elasticsearch"
+      brian_url:                      "http://project-brian:8083"
```
Comment (lindenmckenzie) [inline v2/manifests/core-ons/zebedee.yml:51]: Could you add this to the Zebedee readme as well?

File: v2/stacks/static-files-with-auth/deps.yml [hunk 2]
```diff
@@ -1,28 +1,48 @@
...
     extends:
       file: ${PATH_MANIFESTS}/deps/localstack.yml
       service: localstack
-  http-echo:
+  elasticsearch:
     extends:
-      file: ${PATH_MANIFESTS}/deps/http-echo.yml
-      service: http-echo
+      file: ${PATH_MANIFESTS}/deps/elasticsearch.yml
+      service: elasticsearch
```
Comment (lindenmckenzie) [inline v2/stacks/static-files-with-auth/deps.yml:48]: are you sure this is the elasticsearch you want?

File: v2/stacks/static-files-with-auth/local.env [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# get from cognito: sandbox-florence-users
+AWS_COGNITO_USER_POOL_ID=""
+# get from within pool - App Integration: App client: dp-identity-api
+AWS_COGNITO_CLIENT_ID=""
+# get from within pool - App Integration: App client: dp-identity-api
+AWS_COGNITO_CLIENT_SECRET=""
+
+AWS_ACCESS_KEY_ID=""
+AWS_SECRET_ACCESS_KEY=""
+AWS_SESSION_TOKEN=""
```
Comment (lindenmckenzie) [inline v2/stacks/static-files-with-auth/local.env:28]: no newline?

**Review by lindenmckenzie — APPROVED** [review]
Some minor comments but happy to approve as is.

File: v2/manifests/core-ons/zebedee.yml (hunk 1 above)
Comment (franmoore05) [reply v2/manifests/core-ons/zebedee.yml:51]: Done :)

**Pull request merged**